platform/src/main/java/org/stellar/anchor/platform/configurator/DataConfigAdapter.java

```diff
@@ -0,0 +1,182 @@
+ * </pre>
+ *
+ * Spring JPA.
+ *
+ * <pre>
+ * spring.jpa.database: 	                           Target database to operate on, auto-detected by default. Can be alternatively set using the "databasePlatform" property.
+ * spring.jpa.database-platform:                       Name of the target database to operate on, auto-detected by default. Can be alternatively set using the "Database" enum.
+ * spring.jpa.defer-datasource-initialization:         Whether to defer DataSource initialization until after any EntityManagerFactory beans have been created and initialized.: false
+ * spring.jpa.generate-ddl:                            Whether to initialize the schema on startup. : false
+ * spring.jpa.hibernate.ddl-auto:                      DDL mode. This is actually a shortcut for the "hibernate.hbm2ddl.auto" property. Defaults to "create-drop" when using an embedded database and no schema manager was detected. Otherwise, defaults to "none".
+ * spring.jpa.hibernate.naming.implicit-strategy:      Fully qualified name of the implicit naming strategy.
+ * spring.jpa.hibernate.naming.physical-strategy:      Fully qualified name of the physical naming strategy.
+ * spring.jpa.hibernate.use-new-id-generator-mappings: Whether to use Hibernate's newer IdentifierGenerator for AUTO, TABLE and SEQUENCE. This is actually a shortcut for the "hibernate.id.new_generator_mappings" property. When not specified will default to "true".
+ * spring.jpa.mapping-resources:                       Mapping resources (equivalent to "mapping-file" entries in persistence.xml).
+ * spring.jpa.open-in-view:                            Register OpenEntityManagerInViewInterceptor. Binds a JPA EntityManager to the thread for the entire processing of the request. : true
+ * spring.jpa.show-sql:                                Show SQL : false
+ * </pre>
+ */
+public class DataConfigAdapter extends SpringConfigAdapter {
+  List<String> allFields =
+      Arrays.asList(
+          "spring.datasource.driver-class-name",
+          "spring.datasource.name",
+          "spring.datasource.type",
+          "spring.datasource.url",
+          "spring.datasource.username",
+          "spring.datasource.password",
+          "spring.datasource.generate-unique-name",
+          "spring.datasource.hikari.connection-timeout ",
+          "spring.datasource.hikari.minimum-idle",
+          "spring.datasource.hikari.maximum-pool-size",
+          "spring.datasource.hikari.idle-timeout",
+          "spring.datasource.hikari.max-lifetime",
+          "spring.datasource.hikari.auto-commit ",
+          "spring.jpa.database",
+          "spring.jpa.database-platform",
+          "spring.jpa.defer-datasource-initialization",
+          "spring.jpa.generate-ddl",
+          "spring.jpa.hibernate.use-new-id-generator-mappings",
+          "spring.jpa.open-in-view",
+          "spring.jpa.show-sql",
+          "spring.flyway.enabled");
```

Comment:
> string literals are usually considered code smells.

I know this can be debating. I used to think that way that literals smell. But I also found values in using string literals when it makes the code more clear. 

I actually try to extract literals to constants. It actually makes the code less readable except it prevents typo. Weighing both, I would suggest that we go with literals at this moment.